app/Assets/SizeVariantLegacyNamingStrategy.php

```diff
@@ -39,7 +39,7 @@ public function setPhoto(?Photo $photo): void
 	{
 		parent::setPhoto($photo);
 		$this->originalExtension = '';
-		if ($this->photo && $sv = $this->photo->size_variants->getOriginal()) {
+		if ($this->photo != null && ($sv = $this->photo->size_variants->getOriginal()) != null) {
```

Comment:
```suggestion
		if ($this->photo !== null && ($sv = $this->photo->size_variants->getOriginal()) !== null) {
```

I am not sure, if I like this PhpStan rule. Due to type hinting as nullable objects of a specific class, i.e. `?Photo` and `?SizeVariant` the short syntax is not ambiguous and IMHO more readable. There is only one way how `photo` can be evaluated as a boolean.

But if we need it, we should at least use the strict comparison.